package.json

```diff
@@ -14,6 +14,7 @@
 		"build:plugin": "MODE=CLI ANALYTICS=true npm run build && bitrise run create-binaries",
 		"start:dev": "webpack-dev-server",
 		"start": "concurrently \"npm:start:dev\" \"DEV=true ./_scripts/run_api.sh 4000\"",
+		"start:website": "export MODE=WEBSITE && export PUBLIC_URL_ROOT=http://localhost:4000 && npm run start",
```

Comment:
do we need exports here? 🤔

Reply:
I was wondering too, but I can't get it work the way it works one line above :( If you have any suggestion, I'll iterate an other round here

Reply:
can't we do `MODE=WEBSITE PUBLIC_URL_ROOT=http://localhost:4000 npm start`?

Reply:
hmm, last time I tried this, it didn't work, now it's fine :/ anyway, I fixed it :)